contrib/registry/servicecomb/go.mod

```diff
@@ -0,0 +1,15 @@
+module servicecomb
```

Comment:
The Module name can be defined as `github.com/go-kratos/kratos/contrib/servicecomb/v2`

Reply:
done